pkgs/by-name/iw/iwd/package.nix

```
@@ -13,6 +14,23 @@
   gitUpdater,
 }:
 
+let
+  readline-patch = fetchpatch {
+    url = "https://lists.gnu.org/archive/html/bug-readline/2025-07/txtmA7rksnmmi.txt";
+    hash = "sha256-GIuZWC+dMenBjdPkqQOsX7TI4mUgfPwh/CfzLtTuang=";
```

```suggestion
    hash = "sha256-QSS1GUJ2i/bF2ksvUtw27oqFHuTHALi+7QwxMFt9ZaM=";
    stripLen = 2;
```

This saves us from invoking `patch` in `postPatch`.